ibis/expr/api.py

```diff
@@ -453,18 +456,20 @@ def desc(expr: ir.Column | str) -> ir.Value:
     Examples
     --------
     >>> import ibis
-    >>> t = ibis.table(dict(g='string'), name='t')
-    >>> t.group_by('g').size('count').order_by(ibis.desc('count'))
-    r0 := UnboundTable: t
-      g string
-    r1 := Aggregation[r0]
-      metrics:
-        count: Count(t)
-      by:
-        g: r0.g
-    Selection[r1]
-      sort_keys:
-        desc|r1.count
+    >>> ibis.options.interactive = True
+    >>> t = ibis.examples.penguins.fetch()
+    >>> t[["species", "year"]].order_by(_.year.desc()).head()
```

Comment:
How is this completing if `_` hasn't been imported from `ibis`?

Reply:
Ah, because it's an interactive session, so `_` is `t` :)

That's not intentional, so I'll fix it up.

Nice catch.

Reply:
Fixed.